weathering_pset_student_1.Rmd

```diff
@@ -574,4 +579,3 @@ JHR - Plot accumulation of minerals with reaction progress as area chart - not c
 # ggplotly(plot_minerals_rxns_1_2_3_4_area)
 ```
 
-##STUDENT QUESTION: Using the same phase diagram, draw out the pathway you expect the water chemistry to take if we start with _____ and _____. Draw this on your phase diagram given to you in class and turn it in. 
```

Comment:
I'm not sure what you are asking them to do here. If you want them to pick a different start water chemistry and infer a reaction path, I will just note that the reaction paths change drastically and non-intuitively w/ a change in initial conditions (see fig 2 of Steinnman paper.) So such a question may be too complicated